assets/wire/utils.ts

```diff
@@ -326,7 +326,9 @@ export function shortHighlightedtext(html: string, maxLength = 40) {
 
         lastNode = highlightSpan.nextSibling;
     }
-    return extractedText.join(' ')+(html.length > maxLength ? '...' : ' ');
+    return extractedText.join(' ')
+        .replace(/<img[^>]*src="([^"]*)"[^>]*>/g, ' ')
```

Comment:
I wonder if we shouldn't change the code in the for loop, use there `getTextFromNode(highlightSpan)` instead of using the `highlightSpan.outerHTML` and wrap it in the `span` again, otherwise we might have the same issue with other tags